header.php

```diff
@@ -14,7 +14,7 @@
 <html <?php language_attributes(); ?>>
 <head>
 	<meta charset="<?php bloginfo( 'charset' ); ?>">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Removal of space between values isn't actually required. This is just a bug in the AMP plugin fixed by https://github.com/Automattic/amp-wp/pull/1112

Reply:
I've restored the spaces since Automattic/amp-wp#1112 is closed. Can be tested with AMP plugin v1.0-alpha1.